NEWS.md

```diff
@@ -2,6 +2,8 @@
 
 - Addition of `lpc_burg` and `lpc_levinson` methods to calculate the Linear
   Predictive Coefficients describing a signal.
+- dropping support for julia 0.6
+- removing conv2 in favor of n-dimensional conv function
```

Comment:
This hasn't happened yet, has it?

Anyway, this file is hopelessly outdated. The `lpc_*` functions clearly are not "awaiting next release". Not sure what to do here, but outside the scope of this PR.